ci/uploader_rules.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2017 Mozilla Corporation
+# Contributors: Guillaume Destuynder <kang@mozilla.com>
+
+import argparse
+import glob
+import json
+import logging
+import os
+import sys
+from authzero import AuthZero,AuthZeroRule
```

Comment:
Nit. Missing space in comma separated import.
Replace comma or go one per line. Separate suste, from custom classes and alphabetize.